### PR TITLE
Update distance-type.md

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR updates the Distance Types topic in Semantic Search.

### CODE REVIEW

File: docs/search-views/semantic-search/concepts/distance-type.md [hunk 1]
```diff
@@ -13,10 +13,104 @@ You have several distance type options. The different distance or similarity mea
...
+The inner product (also known as the dot product) between two vectors is the sum of the products of their corresponding components. In the context of semantic search, inner product similarity is a measure of similarity that computes the dot product of the query vector and each document vector. High inner product values indicate high similarity.
+
+Inner product incorporates both vector orientation through cosine similarity and vector magnitudes. Inner product is useful when scale matters, such as search based on weighted vectors. It works well for dense vectors. It's especially useful when dealing with vectors that represent things like user and item embeddings in recommendation systems.
+
+A limitation of inner product is sensitivity to noise and outliers, since large values are squared. Inner product also requires vectors of equal length.
```
Comment (amolth-macrometa) [inline docs/search-views/semantic-search/concepts/distance-type.md]: all the distance types need vectors of equal length; it is not unique to ip

Comment (diana-macrometa) [reply docs/search-views/semantic-search/concepts/distance-type.md]: Is that a Macrometa limitation or a universal limitation? If it's just for Macrometa, we should mention it at the top. If it's a universal, then we may or may not mention it.

Comment (amolth-macrometa) [reply docs/search-views/semantic-search/concepts/distance-type.md]: it's universal

Comment (diana-macrometa) [reply docs/search-views/semantic-search/concepts/distance-type.md]: Removed. Anything else need changing?

**Review by amolth-macrometa — APPROVED** [review]

**Pull request merged**